ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -10,26 +10,33 @@
  */
 public class CassandraModule extends ConfigurableModule<CassandraModule.Config> {
 
-	public static class Config {
+	@Override
+	protected void configure() {
+		bind(CassandraService.class).in(Scopes.SINGLETON);
+		bind(CassandraHealthCheck.class).in(Scopes.SINGLETON);
+		bind(FixedRetryPolicy.class).in(Scopes.SINGLETON);
+	}
 
-		public Config() {
-		}
+	public static class Config {
 
 		JKSConfig truststore;
 		JKSConfig keystore;
-
 		String user;
 		String password;
-
 		String keyspace;
 		String validationQuery = "SELECT * FROM system.schema_keyspaces";
-
 		Boolean shareEventLoopGroup = false;
-
 		String migrationFile = "/migrations/cql.changelog";
 		Boolean autoMigrate = false;
-
 		List<String> seeds;
+		int protocolVersion = 2;
+		int readTimeoutMillis = 5000;
+		int defaultConsistencyLevel = 0;
+		boolean defaultIdempotence = false;
+		boolean retryQuery = false;
```

Comment:
I don't think this is how it should work. We can setup a default policy and allow overrides